manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AutodeskDWGTrueView2023
+PackageVersion: 4.0.19.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://www.softpedia.com/dyn-postdownload.php/9e96122ffbf922445ed57a47deca13f7/6366ed52/29028/0/1
+  InstallerSha256: 3D1EB9AD56044D3980F2B14686327E4CB19D0F7378A8EFECBCC6D5BE8B814F43
+  InstallerSwitches:

```

Comment:
This is not the correct format for installer switches. See the [manifest specification](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.2.0/installer.md#installer-manifest)